utils.go

```diff
@@ -133,86 +166,97 @@ func loadLine(line string, gitattributes map[string][]string) error {
 		return nil
 	} else if len(tokens) != 0 {
 		err := errors.New(".gitattributes: Each line only can have a pair of elements  E.g. path/to/file attribute")
-		log.Println(err.Error())
 		return err
 	}
+
 	return nil
 }
 
-func parseAttributes(attributes map[string][]string) []error {
+func parseAttributes(attributes map[string][]string) (*gitAttributes, []error) {
 	errArray := []error{}
+	gitAttributes := newGitAttributes()
 	for key, values := range attributes {
 		for _, val := range values {
-			err := parseAttribute(key, val)
+			err := gitAttributes.parseAttribute(key, val)
 			if err != nil {
 				errArray = append(errArray, err)
 			}
 		}
 	}
 
-	return errArray
+	return gitAttributes, errArray
 }
 
-func parseAttribute(key string, attribute string) error {
-	var err error
-	switch {
-	case strings.Contains(attribute, "linguist-vendored"):
-		err = processVendorAttr(key, attribute)
-	case strings.Contains(attribute, "linguist-documentation"):
-		err = processDocumentationAttr(key, attribute)
-	case strings.Contains(attribute, "linguist-language="):
-		err = processLanguageAttr(key, attribute)
-	default:
-		err = errors.New(fmt.Sprintf("gitattributes: The matcher %s doesn't exists\n", attribute))
-		log.Printf(err.Error())
+func newGitAttributes() *gitAttributes {
+	gA := gitAttributes{
```

Comment:
`gA` is not too meaningful, and it doesn't make sense to upcase the 'A'.
I'd prefer something like 'attrs' or 'attributes', but since really short names are already used elsewhere, at least use proper casing: `ga`